en_US/LC_MESSAGES/django.po

```diff
@@ -8680,33 +8641,23 @@ msgstr "Oops, your request was unsuccessful"
 msgid "We couldn't perform your request. Perhaps because the action is no longer applicable or you tried to perform an invalid action."
 msgstr "We couldn't perform your request. Perhaps because the action is no longer applicable or you tried to perform an invalid action."
 
-#: kitsune/sumo/jinja2/handlers/403.html:3
+#: kitsune/sumo/jinja2/handlers/403.html:2
 msgid "Access denied"
 msgstr "Access denied"
 
-#: kitsune/sumo/jinja2/handlers/403.html:11
-msgid "You must have cookies enabled to login or register."
-msgstr "You must have cookies enabled to login or register."
-
-#: kitsune/sumo/jinja2/handlers/403.html:14
+#: kitsune/sumo/jinja2/handlers/403.html:9
 msgid "You do not have permission to access this page."
 msgstr "You do not have permission to access this page."
 
-#: kitsune/sumo/jinja2/handlers/403.html:19
-#, fuzzy
-msgid "Firefox Accounts"
-msgstr "Firefox"
-
-#: kitsune/sumo/jinja2/handlers/403.html:20
-#, fuzzy
-msgid "SUMO account"
-msgstr "My Account"
+#: kitsune/sumo/jinja2/handlers/403.html:13
+msgid "Please try to log in / sign up first with"
+msgstr "Please try to log in / sign up first with"
```

Comment:
There might be a few sentences like this, concatenated. I assume it is account, my account, SUMO account. Some may have gender issue not knowing what it is.